invokeai/app/services/image_storage.py

```diff
@@ -71,19 +79,75 @@ def __init__(self, output_folder: str):
                 parents=True, exist_ok=True
             )
 
+    def list(
+        self, image_type: ImageType, page: int = 0, per_page: int = 10
+    ) -> PaginatedResults[ImageResponse]:
+        dir_path = os.path.join(self.__output_folder, image_type)
+        image_paths = glob(f"{dir_path}/*.png")
+        count = len(image_paths)
+
+        # TODO: do all platforms support `getmtime`? seem to recall some do not...
```

Comment:
Modification time is not well-implemented on some platforms. Creation time will give you more consistent results.

Reply:
I thought it was the other way around - will change